### ISSUE TITLE
Token required every time

### ISSUE THREAD

**Issue by haha150** [nbgallery/docker-images#8]
So i am using the base-notebook image and whenever i go to the URL of the server, it asks me for either token or password, now i enter the token and get logged in, but if i now when i go incognito, and want to access the server again it asks me for token or password again...is it possible to avoid having to enter the token or password every time i visit the server? 

I am asking this because if nbgallery wants to run notebooks in jupyter and it requires token or password, it cannot access it...basically  "run in jupyter" wont work.

Basically i am wondering if it is possible for nbgallery to "run in jupyter" without me having to log in to the jupyter server first.

Ive also set the NBGALLERY_CONFIG_TOKEN variable

**Comment by jeffyjefflabs** [comment]
No, it's not currently possible to "run in jupyter" without logging into Jupyter first.  We thought about including the password/token as part of the "environment" on the nbgallery side, but it's probably not a good idea from a security standpoint.

The new image doesn't support NBGALLERY_CONFIG_TOKEN (yet; I should maybe add that).  As a workaround (which you can also use to disable the token) you can run the image something like this to pass options to jupyter itself ([start-notebook.sh](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/start-notebook.sh) is the default docker `CMD` from upstream):

```
docker run -p 8888:8888s nbgallery/base-notebook start-notebook.sh --JupyterApp.token=''
```

**Comment by haha150** [comment]
Thanks.

I actually did disable the tokens since we run the servers internally.
